hikari/guilds.py

```diff
@@ -79,6 +79,7 @@
     from hikari import permissions as permissions_
     from hikari import presences as presences_
     from hikari import voices as voices_
+    from hikari.internal import time as time_
```

Comment:
This can be `from hikari.internal import time`, no need for the alias. The reason why the other imports have it is because they collide with the naming of the arguments, which they don't in this case.